docs/extras.md

```diff
@@ -22,6 +22,7 @@ Pagy comes with a few optional extensions/extras:
 | `navs`                | Add nav_js and combo_nav_js javascript unstyled helpers                                                                                                  | [navs.rb](https://github.com/ddnexus/pagy/blob/master/lib/pagy/extras/navs.rb), [documentation](extras/navs.md)                                              |
 | `overflow`            | Allow for easy handling of overflowing pages                                                                                                             | [overflow.rb](https://github.com/ddnexus/pagy/blob/master/lib/pagy/extras/overflow.rb), [documentation](extras/overflow.md)                                  |
 | `searchkick`          | Paginate `Searchkick::Results` objects efficiently                                                                                                       | [searchkick.rb](https://github.com/ddnexus/pagy/blob/master/lib/pagy/extras/searchkick.rb), [documentation](extras/searchkick.md)                            |
+| `meilisearch`          | Paginate `Meilisearch` results efficiently                                                                                                       | [meilisearch.rb](https://github.com/ddnexus/pagy/blob/master/lib/pagy/extras/meilisearch.rb), [documentation](extras/meilisearch.md)                            |
```

Comment:
This line should respect the alpha sorting
Please, also align the `|` delimiters in the table cells